packages/diagrams/lib/src/list_view.dart

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class ListViewDiagram extends StatelessWidget implements DiagramMetadata {
+  const ListViewDiagram(this.name);
+
+  @override
+  final String name;
+  static const List<String> entries = <String>['A', 'B', 'C'];
+  static const List<int> colorCodes = <int>[600, 500, 100];
+
+  @override
+  Widget build(BuildContext context) {
+    Widget returnWidget;
+    switch (name) {
+      case 'list_view':
+        returnWidget = ListView(
+          padding: const EdgeInsets.all(8.0),
+          children: <Widget>[
+            Container(
+              height: 50,
+              color: Colors.amber[600],
```

Comment:
`colorCodes[0]`?

Also, I think it would be shorter to have:
```
children: <int>[0, 1, 2].map((i) => Container(
  height: 50, 
  color: Colors.amber[colorCodes[i]], 
  child: const Center(child: Text('Entry ${entries[index]}')).toList()
```

It would be even shorter if we define
```
  static final List<Widget> entries = <String>['A', 'B', 'C'].map((t) => const Center(child: Text('Entry $t'))).toList();
  static final List<Color> colors = <int>[600, 500, 100].map((i) => Colors.amber[i]).toList();
```
which can be used in the other 2 places.

Reply:
Thanks for the feedback! These are meant to match the sample code in the API docs, that's why they are a little less conventional. You can see the samples [here](https://master-api.flutter.dev/flutter/widgets/ListView-class.html).